src/app/payment/views/IntegratedCheckoutView/CheckoutView.tsx

```diff
@@ -49,15 +55,41 @@ const CheckoutView = ({
   userAuthComponentRef,
   checkoutViewVariables,
   checkoutViewManager,
+  availableCryptoCurrencies,
+  onCurrencyTypeChanges,
 }: CheckoutViewProps) => {
   const { translate } = useTranslationContext();
   // Those custom hooks should be here.
   // They cannot be moved to the Parent, because it must be wrapped by <Elements> component.
   const stripeSDK = useStripe();
   const elements = useElements();
+  const [isCryptoDropdownOpen, setIsCryptoDropdownOpen] = useState<boolean>(false);
+
+  const onCryptoDropdownToggle = () => {
+    const newCryptoState = !isCryptoDropdownOpen;
+
+    if (newCryptoState) {
```

Comment:
nit: could add directly `!isCryptoDropdownOpen` in the `if` condition

Reply:
Ops. Nice catch